src/routes/map/+page.svelte

```diff
@@ -68,7 +56,7 @@
 	img.hidden {
 		display: none;
 	}
-	:root {
-		--state: 3; /* 1 = java, 2 = rivers, 3 = europe */
+	.map :global(circle) {
+		fill: #d9d9d9;
```

Comment:
```suggestion
		fill: var(--color-primary-500);
```
(or whatever value is the right one.